### PR TITLE
feat: Add PHP set up page & examples

### PR DESCRIPTION
**Pull request by jubianchi**

### CODE REVIEW

File: integrations/examples/calling-guest-functions.md [hunk 1]
```diff
@@ -264,7 +293,7 @@ go test examples/example_exports_function_test.go
 {% endtab %}
 
 {% tab title="Python" %}
-You should be able to run it using the `python exports_function.py` command.
+You should be able to run it using the `python exports_function.php` command.
```
Comment (syrusakbary) [inline integrations/examples/calling-guest-functions.md:296]: I think we don't intended this change :P

File: integrations/examples/exit-early.md [hunk 2]
```diff
@@ -282,10 +330,24 @@ If you want to run the examples from the Wasmer [repository](https://github.com/
 
 ```bash
 git clone https://github.com/wasmerio/wasmer-python.git
-cd wasmer-go
+cd wasmer-python
```
Comment (syrusakbary) [inline integrations/examples/exit-early.md:333]: Revert me

File: integrations/examples/instance.md [hunk 3]
```diff
@@ -308,11 +357,17 @@ instance, err := wasmer.NewInstance(module, importObject)
 {% endtab %}
 
 {% tab title="Python" %}
-```go
+```python
```
Comment (syrusakbary) [inline integrations/examples/instance.md:360]: Revert me

File: integrations/examples/exit-early.md [hunk 4]
```diff
@@ -156,6 +177,19 @@ import_object.register(
...
 {% endtab %}
+
+{% tab title="PHP" %}
+```php
+function earlyExit() {
+    throw new ExitCode(1);
+}
+
+$funcType = Wasm\Type\FuncType::new(new Wasm\Vec\ValType(), new Wasm\Vec\ValType());
+$func = Wasm\Module\Func::new($store, $funcType, 'earlyExit');
```
Comment (syrusakbary) [inline integrations/examples/exit-early.md:188]: Side question: why `Func` is inside `Module` ?
Funcs can be used without a module (they just need a store)

File: integrations/examples/instance.md [hunk 5]
```diff
@@ -308,11 +357,17 @@ instance, err := wasmer.NewInstance(module, importObject)
...
 instance = Instance(module)
 ```
 {% endtab %}
 
+{% tab title="PHP" %}
+```php
+$instance = Wasm\Module\Instance::new($store, $module);
```
Comment (syrusakbary) [inline integrations/examples/instance.md:367]: Why Instance is inside Module?

File: integrations/examples/exit-early.md (hunk 4 above)
Comment (jubianchi) [reply integrations/examples/exit-early.md:188]: I tried to be as close as possible to the spec when naming/organizing things: https://webassembly.github.io/spec/core/syntax/modules.html

My main goal was to not put too much things in the `Wasm` namespace directly. What I can do, is change from `Wasm\Module` to `Wasm\External` to replicate what we have in Rust.

File: integrations/examples/exit-early.md (hunk 2 above)
Comment (jubianchi) [reply integrations/examples/exit-early.md:333]: Will do.

See #100

File: integrations/examples/instance.md (hunk 3 above)
Comment (jubianchi) [reply integrations/examples/instance.md:360]: Will do.

See #100

File: integrations/examples/exit-early.md (hunk 2 above)
Comment (jubianchi) [reply integrations/examples/exit-early.md]: done

File: integrations/php/setup.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
+# Setup your Python environment
```
Comment (syrusakbary) [inline integrations/php/setup.md:1]: Setup your PHP environment

File: integrations/examples/exit-early.md (hunk 4 above)
Comment (jubianchi) [reply integrations/examples/exit-early.md]: Fixed.

_See wasmerio/wasmer-php#141_

File: integrations/examples/instance.md (hunk 5 above)
Comment (jubianchi) [reply integrations/examples/instance.md]: Fixed.

_See wasmerio/wasmer-php#141_

File: integrations/examples/errors.md [hunk 7]
```diff
@@ -197,5 +232,18 @@ go test examples/example_errors_test.go
...
+
+{% hint style="info" %}
+If you want to run the examples from the Wasmer PHP [repository](https://github.com/wasmerio/wasmer-php/) codebase directly, you can also do:
+
+```bash
+git clone https://github.com/wasmerio/wasmer-php.git
+cd wasmer-php
+make EXAMPLE=errors test-doc-examples
+```
+{% endhint %}
```
Comment (bobrik) [inline integrations/examples/errors.md:247]: The page doesn't look right, is there a missing `{% endtab %}` here?

File: integrations/examples/instance.md [hunk 8]
```diff
@@ -191,6 +220,18 @@ wasmBytes = open('./path/to/module.wasm', 'rb').read()
 ```
 {% endtab %}
 
+{% tab title="PHP" %}
+```python
```
Comment (bobrik) [inline integrations/examples/instance.md:224]: `php`?

File: integrations/examples/using-exported-globals.md [hunk 9]
```diff
@@ -164,6 +179,20 @@ assert some_type.mutable == True
 ```
 {% endtab %}
 
+{% tab title="Python" %}
```
Comment (bobrik) [inline integrations/examples/using-exported-globals.md:182]: That's no Python: #112.

**Pull request merged**